app/models/trip.rb

```diff
@@ -56,34 +60,37 @@ def self.beautify_group_by_month
   end
 
   def self.popular_starting_station
-    station_id = joins(:start_station)
-                   .group(:start_station_id)
-                   .count
-                   .max_by { |_k, v| v }
-    Station.find(station_id[0]).name unless station_id.nil?
+    station = joins(:start_station)
+      .select('trips.start_station_id, count(trips.start_station_id) as number')
+      .group('trips.start_station_id')
+      .order('number DESC')
+      .first
+      .try(:start_station_id)
+      Station.find(station).try(:name) unless station.nil?
```

Comment:
I might look some more at this this morning bc having the try and unless isn't perfect.